package.json

```diff
@@ -36,11 +35,12 @@
   ],
   "license": "MIT",
   "main": "dist/strophe.js",
-  "browser": "strophe.js",
   "scripts": {
+    "build": "webpack",
     "lint": "make jshint",
     "clean": "make clean",
-    "doc": "make doc"
+    "doc": "make doc",
+    "prepare": "npm run build"
```

Comment:
I personally prefer yarn, so I think it would a good idea to not force a specific package manager at this point.

Reply:
If you are ok with it I can add this small package as devdependency and use it in the scripts: https://www.npmjs.com/package/yarpm

Reply:
But please note the in the repository is commited only `package-lock.json`, which is used by `npm`, and not `yarn.lock` which is used by yarn. That's why I used `npm`, even though I prefer `yarn` too.